tensorflow/core/kernels/shape_ops.cc

```diff
@@ -414,17 +405,14 @@ REGISTER_KERNEL_BUILDER(Name("Squeeze")
 #endif
```

Comment:
Nit.
#endif  // TENSORFLOW_USE_SYCL
I see that file has a few of these, could you make the change throughout this file?